public/src/css/includes/overview.less

```diff
@@ -158,3 +158,29 @@
   }
 
 }
+
+.linz-overview-footer-actions {
+
+  margin-top: 3rem;
```

Comment:
@allanchau, can you remove this? I think it's too much padding and we don't need it.